tests/scripts/pgocheck/pefile/pefile.py

```diff
@@ -2528,7 +2522,7 @@ def parse_directory_bound_imports(self, rva, size):
                 if name_str:
                     invalid_chars = [
                         c for c in bytearray(name_str) if
-                            chr(c) not in string.printable]
+                            unichr(c) not in string.printable]
```

Comment:
I looked through each place where unichr is used, and I did find one that is (potentially) unsafe. I fixed that in commit df4f4db051b42a2eaf0520dcdfeba2ef04b880e1